src/core_ghidra.cpp

```diff
@@ -100,6 +100,7 @@ static void PrintUsage(const RCore *const core)
 		CMD_PREFIX"s",  "", "# Display loaded Sleigh Languages",
 		CMD_PREFIX"ss", "", "# Display automatically matched Sleigh Language ID",
 		CMD_PREFIX"sd", " N", "# Disassemble N instructions with Sleigh and print pcode",
+		CMD_PREFIX"a", "", "# Enable RAsm and RAnal plugins driven by SLEIGH from Ghidra",
```

Comment:
```suggestion
		CMD_PREFIX"a", "", "# Switch to RAsm and RAnal plugins driven by SLEIGH from Ghidra",
```
I think this is a bit more clear about what it does.